linode/helper/framework_resource_base.go

```diff
@@ -55,6 +59,8 @@ func (r *BaseResource) Configure(
 		return
 	}
 
+	r.Client = GetFwClientWithUserAgent(r.Config.Name+" resource", r.Meta)
```

Comment:
Optional nitpick: Would it make sense to distinguish resources/data sources in the UA using the `data.` prefix?

e.g.
`linode_image` vs `data.linode_image`?

Reply:
Thanks for the suggestion! Just added it.